metal/cpu.h

```diff
@@ -69,7 +69,7 @@ int metal_cpu_get_num_harts();
  * @return The value of the CPU cycle count timer
  */
 inline unsigned long long metal_cpu_get_timer(struct metal_cpu *cpu)
-{ return cpu->vtable->timer_get(cpu); }
```

Comment:
The public metal_cpu_get_timer() API should also be rename to metal_cpu_cycle_count()

Reply:
That breaks public API, should we include the old function and mark it deprecated?